### ISSUE TITLE
template<typename T = DWORD> - DWORD as link and/or ref.

### ISSUE THREAD

**Issue by paule32** [doxygen/doxygen#10439]
I get the following output:

template < typename T = DWORD >

the DWORD is static colored like, and "not" clickable.
Is it possible to make DWORD clickable, and refer to a site, where the DWORD is listed, and explained ?
automatically, not extra code in the source ?

**Comment by albert-github** [comment]
`DWORD` is not a standard C++ entity and as such it is not recognized, where is it defined?
- Can you please attach a, small, self contained example (source+configuration file in a, compressed, tar or zip file) that allows us to reproduce the problem? Please don't add external links as they might not be persistent.
- Please also specify the **full** doxygen version used (`doxygen -v`).

**Comment by paule32** [comment]
I solve it: I created a temporay, second/shadow copy of "windows.h" in my documentatin directory, and fill the data information with: **typedef unsigned int DWORD;**
Then I set the PREDEFINED += DOXYGEN
then I set a #ifdef DOXYGEN makro in temp. windows.h, to get sure, that when doxygen is startet, doxygen found the data information's - if #ifdef DOXYGEN will not found (in normal compile of source files) the doxygen comment informations will not collide with pre-existent typedef's.
So, I have two options:
- compile with doxygen
- compile with gnu c++

as result, I get fine documentation, and fine .obj code.
I know, it is like to re-write the original windows.h - but it is only for parts of the file - in my fall - it is DWORD - a 32 bit value.
